notify/docs/listing-templates.md

```diff
@@ -9,9 +9,23 @@ be sent and what personalisation variables each template requires.
 
 ```filetabs
 file=docs/listing-templates/list.py lang=python label=Python
+file=docs/listing-templates/list.java lang=java label=Java
 file=docs/listing-templates/list.sh lang=bash label=cURL
 ```
 
+
+### Arguments
+---
+`template_type`
+
+**Optional**
+
+If omitted, the method returns all templates. Otherwise you can filter by:
+- email
```

Comment:
Should these be in code blocks to indicate they are values? I originally read this as a sentence: "you can filter by email/sms"

Reply:
I've reworded this